www/front_src/src/Provider/models.ts

```diff
@@ -3,7 +3,8 @@ import { KeyValuePair } from 'ramda';
 export interface User {
   locale: string;
   timezone: string;
-  username: string;
+  name: string;
+  alias: string;
 }
 
 export type UserContext = {
```

Comment:
This should be moved to the ui-context repo, along with the other context-related types